src/main/java/com/github/manolo8/darkbot/core/IDarkBotAPI.java

```diff
@@ -134,6 +134,7 @@ default void setVisible(boolean visible, boolean fullyHideEnabled) {
     }
 
     void handleRefresh();
```

Comment:
this can probably just default to calling handleRefresh with a true, and change GameAPIImpl to have a single method (always with a param)

Reply:
I'm not sure it won't mess up some plugins

Reply:
i don't see how it would mess up plugins, just:
```java
default void handleRefresh() {
  handleRefresh(false);
}
```
it should be the exact same behavior as now